test/unit-tests/switchTo.test.js

```diff
@@ -13,4 +13,10 @@ describe('switchTo', () => {
   it('should throw error if no url specified', async () => {
     await expect(taiko.switchTo()).to.eventually.rejectedWith(TypeError);
   });
+
+  it('should throw error if url is empty', async () => {
+    await expect(taiko.switchTo('')).to.eventually.rejectedWith(
```

Comment:
```suggestion
    await expect(taiko.switchTo(' ')).to.eventually.rejectedWith(
```

Should pass with this. Notice the space. Please trim and check.